handlers/aerospike/aerospike.go

```diff
@@ -165,8 +171,12 @@ func (h Handler) HandleGet(item AerospikeGetSet) {
 				item.errorOut <- errors.New("Data retrieved from AeroSpike was corrupted")
 				break
 			}
+			rawFlags, ok := record.Bins["flags"]
+			if ok {
+				flags = uint32(rawFlags.(int))
```

Comment:
OPT: Make it safer by using the `flags, ok =  uint32(rawFlags.(int))` like for values